Frontend/ui-common/src/Components/RideCompletedCard/View.purs

```diff
@@ -1,66 +1,100 @@
+        [ height MATCH_PARENT
+        , width MATCH_PARENT
+        , orientation VERTICAL
+        ][ topCardView config push
+        , bottomCardView config push
       ]
+      ]
+      , if config.isPrimaryButtonSticky then 
+        relativeLayout
+        [ width MATCH_PARENT
+        , height WRAP_CONTENT
+        , weight 1.0
+        , cornerRadii $ Corners 16.0 true true false false
+        , background Color.white900
+        , layoutGravity "bottom"
+        , gravity BOTTOM
+        , alignParentBottom "true,-1"
+        , padding $ Padding 16 16 16 16
+        ][ PrimaryButton.view (push <<< SkipButtonActionController) (config.primaryButtonConfig)]
+        else linearLayout [][]
     , if config.customerIssueCard.reportIssueView then reportIssueView config push else dummyTextView
     , if config.showContackSupportPopUp then contactSupportPopUpView config push else dummyTextView
   ]
-
+  
 topCardView :: forall w. Config -> (Action -> Effect Unit) ->  PrestoDOM (Effect Unit) w
 topCardView config push =
   linearLayout
-  [ width MATCH_PARENT
-  , height $ V ((screenHeight unit)/ 2)
+  ([ width MATCH_PARENT
+  , height WRAP_CONTENT
   , orientation VERTICAL
   , padding $ Padding 16 16 16 16
-  , gradient $ Linear (if os == "IOS" then 90.0 else 0.0) [Color.black900, Color.black900, Color.pickledBlue, Color.black900]
+  , gradient $ Linear (if os == "IOS" then 90.0 else 0.0) config.topCard.gradient
   , gravity CENTER
-  ][  linearLayout
+  ]<> if config.theme == LIGHT then [stroke $ "1,"<>Color.grey800
+                                    , cornerRadii $ Corners 16.0 false false true true
+                                    ] 
+                                else [])
+  [  linearLayout
       [ width MATCH_PARENT
       , height WRAP_CONTENT
       , gravity RIGHT
       , layoutGravity "right"
+      , margin $ MarginLeft 40
       , padding $ PaddingTop safeMarginTop
-      ][  imageView
+      ][ if config.topCard.topPill.visible then 
+            linearLayout[
+              width MATCH_PARENT
+            , height WRAP_CONTENT
+            , gravity CENTER
+            , margin $ MarginLeft 40
+            ][topPillView config push] else dummyTextView
+        , imageView
           [ height $ V 40
           , width $ V 40
           , accessibility config.accessibility
           , accessibilityHint "Contact Support : Button"
-          , imageWithFallback $ "ny_ic_headphone," <> (getCommonAssetStoreLink FunctionCall) <> "ny_ic_headphone.png"
+          , imageWithFallback if config.theme == LIGHT then ("ny_ic_black_headphone," <> (getCommonAssetStoreLink FunctionCall) <> "ny_ic_black_headphone.png") else ("ny_ic_headphone," <> (getCommonAssetStoreLink FunctionCall) <> "ny_ic_headphone.png")
           , onClick push $ const Support
           ]
       ]
-    , if config.topCard.topPill.visible then topPillView config push else dummyTextView
+    -- Removed temporarily till endride screen redesigned
```

Comment:
Can we resolve all the things, which are temporarily and removed the commented code